runtime/bcverify/staticverify.c

```diff
@@ -876,9 +881,13 @@ checkBytecodeStructure (J9CfrClassFile * classfile, UDATA methodIndex, UDATA len
 			index = info->slot2;
 			info = &(classfile->constantPool[classfile->constantPool[index].slot1]);
 			if (info->bytes[0] == '<') {
-				if ((bc != CFR_BC_invokespecial)
-				|| (info->tag != CFR_CONSTANT_Utf8)
-				|| !J9UTF8_DATA_EQUALS("<init>", 6, info->bytes, info->slot1)
+				if (((bc != CFR_BC_invokespecial)
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+						&& (bc != CFR_BC_invokestatic)
```

Comment:
also need to check if its a VT over here. Basically, if its a Vt and its not `CFR_BC_invokestatic` then its invalid. Otherwise the original check